varcode/vcf.py

```diff
@@ -34,19 +43,39 @@ def load_vcf(filename, reference_path_field='reference'):
     reference_path_field : str, optional
         Name of metadata field which contains path to reference FASTA
         file (default = 'reference')
+
+    ensembl_release : int, optional
+        Which release of Ensembl to use for annotation, by default inferred
+        from the reference path.
     """
+
+    if not isinstance(filename, str):
+        raise ValueError(
+            "Expected filename to be str, got %s : %s" % (
+                filename, type(filename)))
+
     vcf_reader = vcf.Reader(filename=filename)
-    reference_name = vcf_reader.metadata[reference_path_field]
+
+    if not ensembl_release:
+        reference_path = vcf_reader.metadata[reference_path_field]
+        reference_name = infer_reference_name(reference_path)
+        ensembl_release = ensembl_release_number_for_reference_name(
+                reference_name)
+
+    ensembl = EnsemblRelease(release=ensembl_release)
 
     variants = [
         Variant(
-            x.CHROM, x.POS,
-            x.REF, x.ALT[0].sequence,
-            x.INFO)
+            contig=x.CHROM,
+            pos=x.POS,
+            ref=x.REF,
+            alt=x.ALT[0].sequence,
+            info=x.INFO,
+            ensembl=ensembl)
```

Comment:
I'm pretty sure what you have is correct (below), but just double checking -- we don't have to compare x.FILTER to "." also, do we?